sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -134,6 +135,16 @@ def _from_generated(cls, generated):
         )
 
 
+class ProtocolProperties(GeneratedProtocolSettings):
```

Comment:
we should make the customer's life as easy as possible. 
With ProtocolProperties, they have to create SmbSettings and SmbMultichannel. While the actually thing they are setting is only a boolean to indicate If SMB multichannel is enabled. Exposing ProtocolProperties will force them to create several instances in order to setting a boolean value. 
So what we want to do is just give them an option like is_smb_multichannel_enabled (not sure if this name is good enough)
https://github.com/tasherif-msft/azure-sdk-for-python/blob/smb-multichannel/sdk/storage/azure-storage-file-share/azure/storage/fileshare/_generated/models/_models.py#L845-L861

Reply:
As per the discussions earlier today we will maintain a layered approach for the Protocol Settings and remove the `settings` suffix.